### ISSUE TITLE
Israelive News

### ISSUE THREAD

**Issue by Harpper** [cubicle-vdo/xbmc-israel#433]
I have no idea if this can be remedied by the developers . i24 News English has been unavailable for some time and several other news channels have cartoons playing instead of the news (for example BBC World news). is this plugin still maintained?

**Comment by avigdork** [comment]
We check the channels from time to time and fix them if there is a problem. 
You can report us about broken links. 
About BBC world news, I checked 3/5 of the sources working. 
I'll tell Benny to find new links and I'll check if there are changes inside some websites.

**Comment by zevnyc** [comment]
I just checked myself
BBC News (not world) only 1 working stream, the 1st. The 3rd resolved but isn't BBC, it's a He-man movie on loop. 
BBC World, link 3 also He-man, same with MSNBC link 1.
These He-man streams are tricking link checking tools because they DO resolve, giving a false positive.

**Comment by avigdork** [comment]
@zevnyc 
I fix the He-man streams, hope it will hold on.
Try the updated channels list.

**Comment by Harpper** [comment]
CBC News and BBC World News still show the He Man cartoon on at least one stream. i24 Live News English has not worked (message says it is unavailable) for some time. Thanks for trying

**Comment by avigdork** [comment]
@Harpper
I checked it now, your channels list is not updated. 
Inside the addon go to Settings and choose to update lists and guides.

**Comment by Harpper** [comment]
Yes I have tried that a few times and it does not seem to change things. 
Still get the cartoon on some streams (CBC, BBC World)

**Comment by Harpper** [comment]
I updated Channel Lists and TV Guide on a Linux PC and a Win 7 PC. Still have issue with cartoons appearing in various streams and i24 Israel News not working at all

**Comment by Harpper** [comment]
I tried uninstalling LiveIsrael and removing all folders related in my .Kodi directory. Then I reinstalled. Made no difference. I still have no i24 News and still have cartoons on a number of streams. Is there a fix for this in the works?

**Comment by zevnyc** [comment]
i24 has a YouTube live stream, I know yt changes the m3u8 for streams every few hours.
Any chance of using the youtu.be/WkvET-MeWbg link and having the resolver handle it?

**Comment by avigdork** [comment]
@Harpper, can you add a log file. I can't find a channel that still play the cartoon thing.
@zevnyc , thank. I added the youtube source.

**Comment by Harpper** [comment]
@avigdork 
Attached
[kodi.log.zip](https://github.com/cubicle-vdo/xbmc-israel/files/920760/kodi.log.zip)
CBC and BBC still have cartoon streams not sure of other networks

**Comment by avigdork** [comment]
@Harpper 
I checked it in Kodi 17, you need to enable RTMP Input addon (in Settings - Addons - VideoPlayer InputStream.

**Comment by SquigglyDog** [comment]
I enabled RTMP Input addon and it made no difference. Sorry I will wait for the next release of Israelive maybe it will be resolved by then

**Comment by SquigglyDog** [comment]
I seem to have 2 accounts for GitHub. I am also Harpper

**Comment by avigdork** [comment]
Sorry, I cannot see any problem on Windows and Android.
I don't have Linux platform to check it.
Did you try it on Kodi 16 or on other device?

**Comment by SquigglyDog** [comment]
Kodi 17

**Comment by avigdork** [comment]
@SquigglyDog 
Yes, I know you tried it on Kodi 17.
Can you try it on Kodi 16 or on on another device? 
I want to find if this happens only on Kodi 17 - Linux.

**Comment by SquigglyDog** [comment]
OK I have it working without the cartoons on my Windows PC by installing RMTP  addon and updating channels in Israelive settings and restarting Kodi. I am still not able to get it working on the Linux Mint system though

**Comment by SquigglyDog** [comment]
BBC News English has disappeared completely. Is there no official source of this?
